### PR TITLE
Format dates

### PR DESCRIPTION
**Pull request by ryankoch13**
# Summary of Changes

Created array of "dateAttributes" in the constants file. If a credential attribute name matches one of the ones listed, it's value will be formatted as a date. 

# Related Issues

[HOL-59](https://indicio.atlassian.net/browse/HOL-59?atlOrigin=eyJpIjoiMjMwMmJhNzJhZDI0NGVhYjkwODhkNGZiNjcyYjljODciLCJwIjoiaiJ9)

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/components/record/RecordField.tsx [hunk 1]
```diff
@@ -65,13 +66,18 @@ const RecordField: React.FC<RecordFieldProps> = ({
...
+        (
           fieldValue(field)
         ) : (
           <>
             <View style={styles.valueText}>
               <Text style={styles.text} testID={testIdWithKey('AttributeValue')}>
-                {shown ? (field as Attribute).value : Array(10).fill('\u2022').join('')}
+                {shown ? 
+                dateAttributes.includes(field.name!) ?
+                new Date((field).value * 1000).toLocaleDateString('en-CA', dateFormatOptions) :
```
Comment (JamesKEbert) [inline core/App/components/record/RecordField.tsx]:
```suggestion
                new Date((field).value * 1000).toLocaleDateString('en-US', dateFormatOptions) :
```

I would suggest changing to the US format (unless I'm missing something here)

File: core/App/components/record/RecordField.tsx [hunk 2]
```diff
@@ -65,13 +66,18 @@ const RecordField: React.FC<RecordFieldProps> = ({
...
+                (field as Attribute).value 
+                : Array(10).fill('\u2022').join('')}
```
Comment (JamesKEbert) [inline core/App/components/record/RecordField.tsx:80]: I realize this is not your code--but do you know what this is doing here @ryankoch13? As this appears to be hardcoded in some form to 2022, which is soon going to be over

File: core/App/components/record/RecordField.tsx (hunk 1 above)
Comment (ryankoch13) [reply core/App/components/record/RecordField.tsx]: Nope, you're totally right, my brain thought it stood for English California for some reason haha

File: core/App/components/record/RecordField.tsx (hunk 2 above)
Comment (ryankoch13) [reply core/App/components/record/RecordField.tsx:80]: it's for unicode character 2022, it's the little star bullet thing they use

File: core/App/components/record/RecordField.tsx (hunk 1 above)
Comment (ryankoch13) [reply core/App/components/record/RecordField.tsx]: This has now been updated to be a constant imported from the constants file and switched to US.

**Review by sabejensen — APPROVED** [review]
lgtm!~

**Pull request merged**